extensions/pkg/controller/backupentry/backupentry_suite_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func TestBackupentry(t *testing.T) {
+func TestBackupEntry(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Backupentry Suite")
+	RunSpecs(t, "Extensions Controller BackupEntry Suite")
```

Comment:
There are a few occurrences left in extensions that are not named as per this convention.
You can find such occurrences using `RunSpecs\(t, "[^E]` regex.

Reply:
Thanks!